docs/how-to/h-async-replication.md

```diff
@@ -0,0 +1,323 @@
+serverconfig_password_1st=$(juju run-action mysql-1st/0 get-password username=serverconfig --wait | yq '.. | select(. | has("password")).password')
+serverconfig_password_2nd=$(juju run-action mysql-2nd/0 get-password username=serverconfig --wait | yq '.. | select(. | has("password")).password')
+clusteradmin_password_1st=$(juju run-action mysql-1st/0 get-password username=clusteradmin --wait | yq '.. | select(. | has("password")).password')
+clusteradmin_password_2nd=$(juju run-action mysql-2nd/0 get-password username=clusteradmin --wait | yq '.. | select(. | has("password")).password')
+
+cat << EOF
+Cluster names:
+  1st: ${cluster_name_1st}
+  2nd: ${cluster_name_2nd}
+Credentials:
+  1st: root:${root_password_1st}
+  2nd: root:${root_password_2nd}
+  1st: serverconfig:${serverconfig_password_1st}
+  2nd: serverconfig:${serverconfig_password_2nd}
+  1st: clusteradmin:${clusteradmin_password_1st}
+  2nd: clusteradmin:${clusteradmin_password_2nd}
+EOF
+```
+
+## Sync credentials
+It is important to set the first cluster credentials on the second cluster to allow charm operator to continue managing 2nd Cluster after joining the ClusterSet:
+```shell
+juju run-action mysql-2nd/leader set-password username=root password=${root_password_1st} --wait # not really necessary.
+juju run-action mysql-2nd/leader set-password username=clusteradmin password=${clusteradmin_password_1st} --wait
+juju run-action mysql-2nd/leader set-password username=serverconfig password=${serverconfig_password_1st} --wait
+```
+
+## Destroy the 2nd Cluster (to be re-created inside the ClusterSet)
+It is NOT possible to "merge" two different MySQL InnoDB Clusters due to [different GTID](https://dev.mysql.com/doc/mysql-shell/8.0/en/innodb-clusterset-rejoin.html) on both Clusters. We are going to dissolve the second Cluster to re-create it as a clone of the first one:
+```shell
+juju ssh mysql-2nd/leader charmed-mysql.mysqlsh --py -h127.0.0.1 -uclusteradmin -p${clusteradmin_password_1st}
+
+> dba.get_cluster().dissolve() # answer YES to all questions, remember IPs of all cluster members
+```
+
+## Re-create the 2nd Cluster as 1st Cluster clone
+Create the second cluster as a clone copy of the first cluster under the first ClusterSet.
+
+**IMPORTANT**: replace all `FIXME_` labels with a proper values (unit IPs and cluster-name)!
+
+```shell
+juju ssh mysql-1st/leader charmed-mysql.mysqlsh --py -h127.0.0.1 -uclusteradmin -p${clusteradmin_password_1st}
+
+> cs = dba.get_cluster_set()
+> cluster2 = cs.create_replica_cluster("clusteradmin@FIXME_IP_1:3306", "FIXME_LISBON", recoveryProgress=1, timeout=10, recoveryMethod='clone')
+> cluster2.add_instance('clusteradmin@FIXME_IP_2:3306', label="mysql-2nd-1")
+> cluster2.add_instance('clusteradmin@FIXME_IP_3:3306', label="mysql-2nd-2")
+```
+
+## Check the status
+Chech the Juju and MySQL InnoDB Cluster(Set) statuses:
```

Comment:
Fixed Lint warning on Discourse.
P.S. @dragomirp it is a good example why Lint tests should be executed on Docs changes, etc. :-)